keyboards/moon/rules.mk

```diff
@@ -0,0 +1,77 @@
+# Project specific files
+SRC = i2c_master.c pca9555.c matrix.c
```

Comment:
```suggestion
SRC += i2c_master.c pca9555.c matrix.c
```

Reply:
Fixed. Good catch

Reply:
It probably wouldn't break anything, but ... better safe than sorry.